RELEASE_NOTES.md

```diff
@@ -1,7 +1,7 @@
 # Upcoming
 
 - fix: No output when compiling to JavaScript on Windows (https://github.com/dafny-lang/dafny/pull/1824)
-
+- fix: No warning "File contains no code" if module contains a submodule (https://github.com/dafny-lang/dafny/pull/1840)
```

Comment:
Should this be in the release notes?

Reply:
Isn't this the file RELEASE_NOTES.md ? Note how in 1 commit, I was able to predict the pull request number :-)

Reply:
I meant that maybe this should not be mentioned in the release notes?

Reply:
We can always decide to remove it later, right @fabiomadge ?